dll/win32/shell32/brsfolder.cpp

```diff
@@ -307,22 +285,22 @@ static HTREEITEM InsertTreeViewItem( browse_info *info, IShellFolder * lpsf,
 	if (!GetName(lpsf, pidl, SHGDN_NORMAL, szBuff))
 	    return NULL;
 
-	lptvid = SHAlloc( sizeof(TV_ITEMDATA) );
+    lptvid = (LPTV_ITEMDATA)SHAlloc( sizeof(TV_ITEMDATA) );
```

Comment:
What's happening with the indentation?

Reply:
Tab character is used in original source.
I will fix indentation later.